tests/integration/test_retry.py

```diff
@@ -88,7 +97,7 @@ def test_retry_timeout():
                     're': retry.CONNECTION_ERROR_AND_TIMEOUT
                 },
                 ttl=0.005,
-                attempt_times=3,
+                retry_limit=3,
```

Comment:
All these values need to be changed if you're changing the meaning of the argument.